exercises/space-age/.meta/generator/space_age_cases.rb

```diff
@@ -0,0 +1,31 @@
+class SpaceAgeCase < Generator::ExerciseCase
+  using Generator::Underscore
+
+  SECONDS_PER_YEAR_ON_EARTH = 31557600
+
+  def workload
+    [
+      "age = SpaceAge.new(#{seconds.underscore})",
+      assertion,
+    ].join("\n")
+  end
```

Comment:
There might be a method in Generator::Exercise case that can join and indent for you.